client/state/ui/selectors.js

```diff
@@ -67,6 +67,7 @@ export const getGuidesTourState = createSelector(
 		const { shouldReallyShow, stepName = '' } = tourState;
 		const stepConfig = getToursConfig()[ stepName ] || false;
 		return Object.assign( {}, tourState, {
+			stepName,
```

Comment:
This is supposed to be [present in the raw state](https://github.com/Automattic/wp-calypso/blob/add/guides-tracking/client/state/ui/reducer.js#L87), no?